packages/web-components/src/menu/menu-split.stories.ts

```diff
@@ -0,0 +1,180 @@
+import { html, repeat } from '@microsoft/fast-element';
+import { type Meta, renderComponent, type StoryArgs, type StoryObj } from '../helpers.stories.js';
+import { ButtonAppearance, ButtonShape, ButtonSize } from '../button/button.options.js';
+import type { Menu as FluentMenu } from './menu.js';
+
+type Story = StoryObj<FluentMenu>;
+
+const defaultSlottedContent = html`
+  <fluent-menu-list style="inset-area: block-end span-inline-start;">
+    <fluent-menu-item>Menu item 1</fluent-menu-item>
+    <fluent-menu-item>Menu item 2</fluent-menu-item>
+    <fluent-menu-item>Menu item 3</fluent-menu-item>
+    <fluent-menu-item>Menu item 4</fluent-menu-item>
+  </fluent-menu-list>
+`;
+
+const generatePimaryActionSlottedContent = (content: string = 'Primary Action') => {
+  return html`<fluent-button
+    appearance="${story => story.appearance}"
+    shape="${story => story.shape}"
+    size="${story => story.size}"
+    slot="primary-action"
+    >${content}</fluent-button
+  >`;
+};
+
+const defaultTriggerSlottedContent = html`<fluent-menu-button
+  appearance="${story => story.appearance}"
+  shape="${story => story.shape}"
+  size="${story => story.size}"
+  slot="trigger"
+  icon-only
+>
+</fluent-menu-button>`;
+
+const storyTemplate = html<StoryArgs<FluentMenu>>`
+  <fluent-menu split>
+    ${story => story.primaryActionSlottedContent?.()} ${story => story.triggerSlottedContent?.()}
+    ${story => story.slottedContent?.()}
+  </fluent-menu>
+`;
+
+export default {
```

Comment:
Maybe we should also have a story for Split Button under Button, to point people to this page, so that our Storybook's left navigation is more aligned with the Figma spec.

Reply:
I don't think we should have the same story in 2 places. But if folks think it's better I can move this story under the `button` category.

Reply:
Sorry I didn't mean to duplicate the stories from the menu one to the split-button's. I just meant, for the split-button's story page, you can have a link to the menu one's page, something like:

```ts
const storyDescription = `
The SplitButton component is implemented as a variation of the Menu component, refer to the Menu → SplitButton page for details.
`;

export default {
  title: 'Components/Button/SplitButton',
  parameters: {
    docs: {
      description: {
        component: storyDescription,
      },
    },
  },
};
```

I think you can link to that story with a Storybook addon but I'm not sure if it's enabled by default with our config.

Reply:
We discussed offline that we will create a story under `src/split-button`. I will add a story to point folks to Menu Stories